packages/backend/jest.env.js

```diff
@@ -1,4 +1,4 @@
-process.env.LOG_LEVEL = 'silent'
+process.env.LOG_LEVEL = 'info'
```

Comment:
maybe we leave it as `silent` to keep the test output tidy

Reply:
Switched that for debugging, meant to change it back before committing.